### PR TITLE
PSR-12 code style fix

### PR DESCRIPTION
**Pull request by alexndr-novikov**
- added spiral/code-style package to require-dev
- PHP version locked in composer up to ^7.2
- removed 7.2 from travis, replaced phpcs PSR-2 code check with spiral/code-style PSR-12

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 7.1
   - 7.2
   - 7.3
```
Comment (wolfy-j) [inline .travis.yml:7]: 7.4snapshot

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
     "phpunit/phpunit": "~7.0",
     "mockery/mockery": "^1.1",
     "squizlabs/php_codesniffer": "^3.4",
-    "friendsofphp/php-cs-fixer": "^2.15"
+    "friendsofphp/php-cs-fixer": "^2.15",
```
Comment (wolfy-j) [inline composer.json]: do we need this line as well?

Comment (alexndr-novikov) [reply composer.json]: removed both lines, code sniffer and fixer

File: .travis.yml (hunk 1 above)
Comment (alexndr-novikov) [reply .travis.yml:7]: added, finished successfully https://travis-ci.org/spiral/boot/builds/595664963

**Pull request merged**